lib/beaker/dsl/install_utils/pe_utils.rb

```diff
@@ -483,6 +503,22 @@ def do_install hosts, opts = {}
           end
         end
 
+        # Helper for setting up pe_defaults & setting up the cert on the host
+        # @param [Host] host                            host to setup
+        # @param [Host] master                          the master host, for setting up the relationship
+        # @param [Array<Fixnum>] acceptable_exit_codes  The exit codes that we want to ignore
+        #
+        # @return nil
+        # @api private
+        def setup_defaults_and_config_helper_on(host, master, acceptable_exit_codes=nil)
```

Comment:
It now only runs on osx/eos & aio.  I figured that since we're duplicating, better to pull it out.

Yes, `acceptable_exit_codes` works correctly.  In the [`host.exec` line it's used in](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host.rb#L272), it's checked, and a default is used if it's falsy, which it will be when set to `nil`.

The default value when the `acceptable_exit_codes` is falsy is `[0, nil]`, and has been for some time, so I doubt that would be a problem.